.github/workflows/bump-tag-publish.yaml

```diff
@@ -88,12 +88,11 @@ jobs:
           VERSION_LINE_MATCH: "^gradle.ext.tanagraVersion\\s*=\\s*\".*\""
           VERSION_SUFFIX: SNAPSHOT
 
-      - name: Set up AdoptOpenJDK
-        if: steps.skiptest.outputs.is-bump == 'no'
```

Comment:
What is the reason for removing this condition? This is to avoid an endless GHA chain where each version bump triggers a new version bump -- even if the other steps are skipped, no need to setup a JDK if we won't use it.

Reply:
oops. this was a copy-paste error. added back